Cargo.toml

```
@@ -41,7 +41,7 @@ base64 = { version = "0.22.0" }
 zeroize = { version = "1.7" } # Not used in serenity, but bumps the minimal version from secrecy
 arrayvec = { version = "0.7.4", features = ["serde"] }
 serde_cow = { version = "0.1.0" }
-small-fixed-array = { version = "0.4", features = ["serde"] }
+small-fixed-array = { version = "0.4, >=0.4.10", features = ["serde"] }
```

```suggestion
small-fixed-array = { version = "0.4.10", features = ["serde"] }
```
This is equal and simpler.